src/js/components/Menu.js

```diff
@@ -170,17 +170,16 @@ class MenuDrop extends Component {
   }
 
   render () {
-    let { dropAlign, size, control, id, colorIndex, onClick } = this.props;
-    let boxProps = Props.pick(this.props, Object.keys(Box.propTypes));
-    // manage colorIndex at the outer menuDrop element
-    delete boxProps.colorIndex;
-
-    delete boxProps.onClick;
-
-    delete boxProps.size;
+    const {
+      dropAlign, size, children, control, id, colorIndex, onClick, ...props
+    } = this.props;
+    delete props.history;
```

Comment:
I wonder if we can make this dynamic using MenuDrop.childContextTypes.

Probably

`Props.omit(props, MenuDrop.MenuDrop.childContextTypes)`